itest/lnd_route_blinding_test.go

```diff
@@ -1510,3 +1510,83 @@ func testBlindedPaymentHTLCReForward(ht *lntest.HarnessTest) {
 		require.Fail(ht, "timeout waiting for sending payment")
 	}
 }
+
+// testGivenBlindedRouteInvoices tests lnd's ability to create a blinded
+// payment path when the income node is given. Which it then inserted into
+// an invoice, sent to an invoice with a blinded path and forwarded payment
+// in a blinded route and finally, receiving the payment.
+func testGivenBlindedRouteInvoices(ht *lntest.HarnessTest) {
+	var (
+		// Convenience aliases.
+		alice = ht.Alice
+		bob   = ht.Bob
+	)
+
+	// Setup a two hop channel network: Alice -- Bob -- Carol.
```

Comment:
maybe you can reuse: `createThreeHopNetwork`

Reply:
let's keep it as is for now without using `createThreeHopNetwork` need to think about a place where to put common functions for itest.

Reply:
ok I found what I was looking for use: `ht.CreateSimpleNetwork`

Reply:
Great ... thank you!